miniapp/src/main/assets/bridge.js

```diff
@@ -15,22 +16,24 @@ var isPlatform = {
  * Method to call the native interface methods for respective platforms
  * such as iOS & Android
  * @param  {[String]} action Action command/interface name that native side need to execute
+ * @param  {Object} param Object that contains request parameter values like permissions.
+ * For eg., {permission: 'location'}
  * @param  {[Function]} onSuccess Success callback function
  * @param  {[Function]} onError Error callback function
  */
-MiniAppBridge.exec = function(action, onSuccess, onError) {
+MiniAppBridge.exec = function(action, param, onSuccess, onError) {
```

Comment:
Can we follow the convention lines of JSON RPC instead (`method`, `params`)?
@ts-leojoseph-r @corycaywood @minh-rakuten

Reply:
That's basically what we're doing, so it's really just a matter of naming, right?

Reply:
Yeah, that's right. We send the request like a JSON RPC already

Reply:
Yes, primarily the naming in this turn. Structure uniformity we could probably target in the next ones. What do you guys think?

Thanks @ts-leojoseph-r for attempting based on the historical discussion.